integration-tests/bats/import-update-tables.bats

```diff
@@ -1017,3 +1017,149 @@ DELIM
     [ "$status" -eq 1 ]
     [[ "$output" =~ "All constraints are not satisfied" ]] || false
 }
+
+@test "import-update-tables: bit types" {
+    dolt sql -q "CREATE TABLE bitted (id int PRIMARY KEY, b bit)"
+    dolt sql -q "INSERT INTO bitted VALUES (1, 0), (3, 1)"
+
+    dolt table export bitted bitted.csv
+
+    run dolt table import -u bitted bitted.csv
+    [ "$status" -eq 0 ]
+    [[ "$output" =~ "Rows Processed: 2, Additions: 0, Modifications: 0, Had No Effect: 2" ]] || false
+
+    run dolt sql -r csv -q "select * from bitted order by id"
+    [[ "$output" =~ "id,b" ]] || false
+    [[ "$output" =~ "1,0" ]] || false
+    [[ "$output" =~ "3,1" ]] || false
+
+    # Try with a larger bit size
+    dolt sql -q "create table bitted2 (id int PRIMARY KEY, b bit(4))"
+    dolt sql -q "INSERT INTO bitted2 values (1, 4)"
+
+    dolt table export -f bitted2 bitted.csv
+
+    run dolt table import -u bitted2 bitted.csv
+    [ "$status" -eq 0 ]
+    [[ "$output" =~ "Rows Processed: 1, Additions: 0, Modifications: 0, Had No Effect: 1" ]] || false
+
+    run dolt sql -r csv -q "select * from bitted2 order by id"
+    [ "$status" -eq 0 ]
+    [[ "$output" =~ "id,b" ]] || false
+    [[ "$output" =~ "1,4" ]] || false
+
+    # Try with a binary value like 0x04
```

Comment:
(minor) technically `0x04` is hex, which is great to test. Would be good to include a binary representation, too.

Reply:
you *might* even consider using a hex number that doesn't map directly to it's base-10 number, like `0xa` or `0x10` (I know that doesn't currently fit in the current bit width though.) That would just test a _little_ bit more of the hex case and give us a sanity check that we are handling it as hex.

Reply:
Sure! I did add binary representation in the form b'100' below.

Reply:
Added 0xa